### PR TITLE
License Bulk Registration

### PR DESCRIPTION
**Pull request by Gseungmin**
Many licenses you want can be registered at once

Signed-off-by: Ji Seungmin <cswcsm02@gmail.com>

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [ ] Refactoring, Maintenance
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: src/main/java/oss/fosslight/controller/LicenseController.java [hunk 1]
```diff
@@ -232,8 +231,9 @@ public String edit(@PathVariable String licenseId, HttpServletRequest req, HttpS
 
 		if(!isNew) {
 			beforeBean =  licenseService.getLicenseMasterOne(licenseMaster);
+			System.out.println("beforeBean = " + beforeBean);
```
Comment (soimkim) [inline src/main/java/oss/fosslight/controller/LicenseController.java:234]: @Gseungmin , 테스트용 출력은 삭제해주세요~

File: src/main/java/oss/fosslight/controller/LicenseController.java [hunk 2]
```diff
@@ -476,6 +476,217 @@ public String LicenseBulkRegPage(HttpServletRequest req, HttpServletResponse res
...
+			, HttpServletRequest req
+			, HttpServletResponse res
+			, Model model) {
+		List<Map<String, Object>> licenseDataMapList = new ArrayList<>();
+		Map<String, Object> resMap = new HashMap<>();
+
+		if (licenseMasters.isEmpty()) {
+			//When the licenseMaster List delivered from the client is empty (if the upload button is pressed without uploading the file)
+			resMap.put("res", false);
+			System.out.println("licenseMaster is Empty");
```
Comment (soimkim) [inline src/main/java/oss/fosslight/controller/LicenseController.java:495]: @Gseungmin , 테스트용 출력은 삭제해주세요~

Comment (Gseungmin) [reply src/main/java/oss/fosslight/controller/LicenseController.java]: 삭제했습니다!

File: src/main/java/oss/fosslight/controller/LicenseController.java (hunk 1 above)
Comment (Gseungmin) [reply src/main/java/oss/fosslight/controller/LicenseController.java]: 삭제했습니다!

**Comment by soimkim** [comment]
@Gseungmin 님~! Awesome합니다. 👍

**Pull request merged**